WordPress/gutenberg
# Multi-selection: avoid RichText instances becoming disabled after multi selection.

## Description

Currently there's a bug where rich text instances become disabled after multi selection. I haven't found a way to consistently reproduce this, but the point is that the way we disable rich text instance for multi-selection and re-enable them afterwards is quite fragile.

There's two ways to get a native selection to work across instances:

1. Removing the `contenteditable` attribute will allow you to expand selection outside of the borders of the currently focussed rich text instance. This is what we currently do.
2. Having a temporary `contenteditbale` wrapper around all rich text instances will natively allow you to expand selection outside of the borders of the currently focussed nested rich text instance.

This PR explores the second approach, which would also be beneficial for future cross block selection.

The new approach is much simpler because instead of having to change attributes for every single rich text instance on the page, we only have to change the attribute on the writing flow wrapper, which should also perform better.

## Testing Instructions

I find the bug is most easily reproducible in Safari. Open the demo content and multi select some blocks. Then delete the multi selection by clicking on another block. Check blocks further down the page to see if it's possible to place the caret in them. Some rich text areas will still be disabled. You may have to try several times to reproduce the issue.

With this PR, and following the steps above, the issue should be gone.

## Screenshots 

## Types of changes





## Checklist:
